src/tiled/mainwindow.cpp

```diff
@@ -791,13 +883,13 @@ void MainWindow::export_()
     QString exportFileName = mapDocument->lastExportFileName();
 
     if (!exportFileName.isEmpty()) {
-        MapFormat *exportFormat = mapDocument->exportFormat();
+        MapFormat *exportFormat = qobject_cast<MapFormat*>(mapDocument->exportFormat());
```

Comment:
Cast not needed because `MapDocument::exportFormat` returns `MapFormat*`.

Reply:
Done